code/pilotfile/csg.cpp

```diff
@@ -966,6 +966,7 @@ void pilotfile::csg_write_redalert()
 void pilotfile::csg_read_hud()
 {
 	int idx;
+	int strikes = 0;
```

Comment:
This is not used in this function but should probably be incremented with every error correction (like in the other function).